### DISCUSSION TITLE
[Web|Mobile|Desktop] Make defaultEmbedBuilder visible outside of package

### DISCUSSION THREAD

**Discussion by 5turman** [singerdmx/flutter-quill#704] [General]
To reuse defaultEmbedBuilder in my custom builder I have to use direct import:
`import 'package:flutter_quill/src/widgets/embeds/default_embed_builder.dart';`

Linter gives me warning about such import.

**Comment by kairan77** [comment]
You would be better off extracting the deaultEmbedBuilder file from source, rename and copy to your own workspace,  there are quite a bit of other issues with that file, you could clean up the code and fix those issues in your own version and make it fit better with your need.

I suspect they don't consider deaultEmbedBuilder a core asset for this package, it is probably there for demo and is meant to be overwritten by users of this package anyway.

**Comment by xinyuezhang0402** [comment]
We consider deaultEmbedBuilder a core asset for this package.
Please submit PR to fix any issue if there is any
